plugins/package-managers/go/src/funTest/kotlin/GoModFunTest.kt

```diff
@@ -35,9 +35,9 @@ import org.ossreviewtoolkit.utils.test.matchExpectedResult
 class GoModFunTest : StringSpec({
     val testDir = getAssetFile("projects/synthetic/gomod")
 
-    "Project dependencies are detected correctly" {
+    "Project dependencies are detected correctly given a project with test dependencies" {
         val definitionFile = testDir.resolve("gomod/go.mod")
-        val expectedResultFile = testDir.resolve("gomod-expected-output.yml")
+        val expectedResultFile = testDir.resolve("project-with-test-deps-expected-output.yml")
```

Comment:
Hmm, the directory is now called "project-with-tests", so shouldn't this omit "deps-" and be named "project-with-tests-expected-output.yml"? Also, in line 39 above, shouldn't it be "project-with-tests/go.mod"?